ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -392,4 +384,50 @@ public FrontendMetrics(MetricRegistry metricRegistry) {
     putWithAccountAndContainerHeaderRate = metricRegistry.meter(
         MetricRegistry.name(AmbryBlobStorageService.class, "PutWithAccountAndContainerHeaderRate"));
   }
+
+  /**
+   * Class to hold different {@link RestRequestMetrics} for plain text, ssl, with and without encryption
+   */
+  class RestRequestMetricsGroup {
+    private final RestRequestMetrics blobMetrics;
+    private final RestRequestMetrics blobSSLMetrics;
```

Comment:
minor: I used the wrong capitalization on these variables. Could we change these to "Ssl" instead of "SSL". That will better match the actual metric name.